.github/workflows/dev.yml

```diff
@@ -473,14 +473,11 @@ jobs:
           submodules: true
 
       - name: Build
-        run: zig build
+        run: zig build -DJIT
```

Comment:
Actually, could we rename `-DJIT` to `-DSUPPORT_JIT`, so that the zig build options match CMake? (Suggestion)